package.json

```diff
@@ -71,9 +71,11 @@
     "@polkadot/types": "1.34.1",
     "@polkadot/util": "3.4.1",
     "@polkadot/util-crypto": "3.4.1",
+    "assert": "^2.0.0",
```

Comment:
im pretty sure polkadot has some assert lib, maybe worth using that instead of this

Reply:
Removed the `assert` dependency in favor of polkadot's version. d9b4eba